### PR TITLE
Selenium: Improve and rework the 'FetchUpdatesAndMergeRemoteBranchIntoLocalTest'

### PR DESCRIPTION
**Pull request by artaleks9**
### What does this PR do?
* Improve and rework to use `kohsuke Github library` the selenium test _FetchUpdatesAndMergeRemoteBranchIntoLocalTest_ related to the issue #8875
* Rename the _FetchUpdatesAndMergeRemoteBranchIntoLocalTest_ to the _FetchAndMergeRemoteBranchToLocalTest_
* Move the _FetchUpdatesAndMergeRemoteBranchIntoLocalTest_ from the _CheOneThreadTestsSuite.xml_ to the _CheSuite.xml_

### What issues does this PR fix or reference?
#8875, #8153

### CODE REVIEW

**Comment by dmytro-ndp** [comment]
"merge from one branch **into** another" is a correct phrase, when we are talking about merging.

**Comment by dmytro-ndp** [comment]
Do we still need https://github.com/eclipse/che/blob/6.1.0/selenium/che-selenium-test/src/test/resources/projects/testRepo-1/src/main/java/commenttest/FetchUpdatesAndMergeRemoteBranchIntoLocalTest.java resource?

**Comment by dmytro-ndp** [comment]
ci-test

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/FetchAndMergeRemoteBranchIntoLocalTest.java [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+    git.importJavaApp(repoUrl, PROJECT_NAME, MAVEN);
+
+    // change content in the test repo on GitHub
+    deleteFileOnGithubSide(String.format("%s/%s", pathToJspFile, jspFile), "delete index.jsp");
+    changeContentOnGithubSide(
+        String.format("%s/%s.java", pathToJavaFile, javaFile), CHANGE_CONTENT);
+    changeContentOnGithubSide(textFile, CHANGE_CONTENT);
+
+    performFetch();
+
```
Comment (musienko-maxim) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/FetchAndMergeRemoteBranchIntoLocalTest.java]: How do we know that fetch has been performed as well?

**Comment by artaleks9** [comment]
Add the check of the mesage in the console `git fetch` is preformed well

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/FetchAndMergeRemoteBranchIntoLocalTest.java (hunk 1 above)
Comment (artaleks9) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/FetchAndMergeRemoteBranchIntoLocalTest.java]: Add the check of the message in the console that `git fetch` is preformed well

**Review by dmytro-ndp — APPROVED** [review]

**Pull request merged**